.github/workflows/lint_doc.yml

```diff
@@ -1,83 +0,0 @@
-name: Lint Docs
```

Comment:
I think we want to keep this test to avoid drift in our docs, I couldn't find it in the new approach

Reply:
Let's do it as a follow-up https://github.com/cloudquery/cloudquery/issues/2138